packages/modelence/src/app/index.ts

```diff
@@ -154,6 +158,10 @@ function getConfigSchema(modules: Module[]): ConfigSchema {
         );
       }
 
+      if (key.toLowerCase().startsWith('_system.')) {
+        throw new Error(`Config key cannot start with a reserved prefix: '_system.' (${key})`);
+      }
+
```

Comment:
I think this won't work because we already namespace configs with the corresponding module name. So currently if you have a module called `_system.auth`, it would just need to pass `google.apiKey` as the config name (see the `absoluteKey` calculation above).